spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'machinist/active_record'
+require 'polyamorous/polyamorous.rb'
```

Comment:
We need the specs from the Polyamorous gem added and passing.

Reply:
ok

Reply:
Hi @seanfcarroll - All files are covered with the specs and coverage is 100%, let me know the file and line which is not covered with the spec.

Thanks

Reply:
Could you please add in the code missing from here https://github.com/activerecord-hackery/polyamorous/blob/master/spec/polyamorous/join_association_spec.rb#L20-L37

Also, please move the Polyamorous tests under their own folder (at the same level as Ransack).

Thank you. :-)